index.html

```diff
@@ -3177,6 +3177,273 @@ <h3>Definition of Roles</h3>
 						</tbody>
 					</table>
 				</div>
+        <div class="role">
+          <rdef>doc-pagefooter</rdef>
+          <div class="role-description">
+            <p>A repeated section of text that is separated from the body text and appears at the bottom of a page, containing items such as the page number, footnotes, and repeated items such a shortened document title or the author's last name.</p>
```

Comment:
Footnotes aren't traditionally considered a part of the running footer, so this could lead to misunderstanding about how this role is supposed to be used (e.g., that it can be used on any HTML header or footer element for common preliminary/trailing matter in a chapter/section). It might be better to focus on the repeating template used, as incrementing page numbers isn't new content but just a part of that pattern.

Perhaps wording more like the following:

"A section of text appearing at the bottom of a page that provides context about the current work and location within it. The page footer is distinct from the body text and normally follows a repeating template that contains (possibly truncated) items such as the document title, current section, author name(s), and page number."

Reply:
Thanks for the suggested wording. I'll use it.

Good to know. Maybe we should ask for an edit in the Wikipedia article for page footer, which says:
"The footer is also sometimes the space reserved for the notes, known as footnotes. These footnotes are distinguished from the notes placed at the end of a chapter or book. While in some cases the footer space is generally fixed, when footnotes are added, the space will increase or decrease based on the amount and length of the notes."